### DISCUSSION TITLE
How can I use the Zustand in Angular??

### DISCUSSION THREAD

**Discussion by Syammed2429** [pmndrs/zustand#2492] [Q&A] [answered]
Does zustand supports Angular?
How can I use the Zustand in Angular??
Are there any documentation or guides to implement??

**Comment by dbritto-dev** [comment, marked as answer]
@Syammed2429 unfortunately, zustand does not support angular. Angular needs of rxjs so you have to right your own adapter or use ngx-zustand package

**Reply by Syammed2429** [reply]
So I can use ngx-zustand right

**Reply by dbritto-dev** [reply]
@Syammed2429 yes

**Comment by kamalkech** [comment]
@Syammed2429 i not recommend to use ngx-zustand cuz it s not an official package of zustand cuz probably u will get errors on next version of angular and it s just an adapter package

**Reply by Syammed2429** [reply]
> @Syammed2429 i not recommend to use ngx-zustand cuz it s not an official package of zustand cuz probably u will get errors on next version of angular and it s just an adapter package 

What state management do you suggest for the angular?

**Reply by kamalkech** [reply]
> > @Syammed2429 i not recommend to use ngx-zustand cuz it s not an official package of zustand cuz probably u will get errors on next version of angular and it s just an adapter package
> 
> What state management do you suggest for the angular?

ngrx store
https://ngrx.io/guide/store
